src/implementations/vanilla/components/text-area/_text-area.scss

```diff
@@ -0,0 +1,110 @@
+/**
+ * Textarea component
+ * @define text-area
+ */
+
+@use "sass:map";
+@use '@ecl/theme-dev/theme';
+@use '@ecl/vanilla-layout-grid/mixins/breakpoints';
+
```

Comment:
This is also in the from group, but from what i see in the specs the label font-size is different than the one for the span (optional), in EC only, just mentioning this, i'm getting sick in seeing this minimal differences in the specs for the two systems, actually...